cmd/vic-machine/create/create.go

```diff
@@ -176,6 +198,166 @@ func (c *Create) Flags() []cli.Flag {
 	return flags
 }
 
+func zeroIPNet(n net.IPNet) bool {
```

Comment:
There is a reflect Value.IsNil function - suggest naming this similarly as the intent is identical, just not applied to a reflect.Value